### PR TITLE
Simplifying DebugSession>>isContextPostMortem:

### PR DESCRIPTION
**Pull request by dupriezt**
Issue #2927

### CODE REVIEW

**Comment by dupriezt** [comment]
Failing tests:
- osx-32 / Tests-osx-32 / testChangeWidgetTextUpdatesPresenter – MacOSX32.Spec.MorphicBackendTests.MorphicTextInputFieldAdapterTest<1s
- osx-32 / Tests-osx-32 / testMaxLengthIsSetInWidget – MacOSX32.Spec.MorphicBackendTests.MorphicTextInputFieldAdapterTest<1s
- windows-32 / Tests-windows-32 / testChangeWidgetTextUpdatesPresenter – Windows32.Spec.MorphicBackendTests.MorphicTextInputFieldAdapterTest<1s
- windows-32 / Tests-windows-32 / testMaxLengthIsSetInWidget – Windows32.Spec.MorphicBackendTests.MorphicTextInputFieldAdapterTest<1s

File: src/DebuggerModel/DebugSession.class.st [hunk 1]
```diff
@@ -211,9 +211,7 @@ DebugSession >> isContextPostMortem: selectedContext [
 	| suspendedContext |
 	suspendedContext := interruptedProcess suspendedContext.
 	suspendedContext ifNil: [ ^ false ].
-	(suspendedContext == selectedContext)
-		ifTrue: [ ^ false ].
-	^ (suspendedContext findContextSuchThat: [:c | c sender == selectedContext]) isNil
+	^ (suspendedContext hasContext: selectedContext) not.
```
Comment (guillep) [inline src/DebuggerModel/DebugSession.class.st:214]: It looks good, but is there a test for this functionality?

**Review by guillep — CHANGES_REQUESTED** [review]

File: src/DebuggerModel/DebugSession.class.st (hunk 1 above)
Comment (dupriezt) [reply src/DebuggerModel/DebugSession.class.st:214]: Yes, in IsContextPostMortemTests>>#testIsContextPostMortem

Comment (guillep) [reply src/DebuggerModel/DebugSession.class.st:214]: ok!

**Review by guillep — APPROVED** [review]

**Pull request merged**